core/src/com/unciv/ui/popup/Popup.kt

```diff
@@ -109,51 +110,38 @@ open class Popup(
     }
 
     /**
-     * Adds an inline [TextButton].
+     * Adds a [TextButton] and ends the current row.
      * @param text The button's caption.
      * @param key Associate a key with this button's action.
      * @param action A lambda to be executed when the button is clicked.
      * @return The new [Cell]
      */
-    fun addButtonInRow(text: String, key: KeyCharAndCode? = null, action: () -> Unit): Cell<TextButton> {
-        val button = text.toTextButton()
+    fun addButton(text: String, key: KeyCharAndCode? = null, style: TextButtonStyle? = null, action: () -> Unit): Cell<TextButton> {
+        val button = text.toTextButton(style)
         button.onActivation { action() }
         button.keyShortcuts.add(key)
         return add(button)
     }
-    fun addButtonInRow(text: String, key: Char, action: () -> Unit)
-        = addButtonInRow(text, KeyCharAndCode(key), action)
-    fun addButtonInRow(text: String, key: Int, action: () -> Unit)
-        = addButtonInRow(text, KeyCharAndCode(key), action)
-
-    /**
-     * Adds a [TextButton] and ends the current row.
-     * @param text The button's caption.
-     * @param key Associate a key with this button's action.
-     * @param action A lambda to be executed when the button is clicked.
-     * @return The new [Cell]
-     */
-    fun addButton(text: String, key: KeyCharAndCode? = null, action: () -> Unit)
-        = addButtonInRow(text, key, action).apply { row() }
     /** @link [addButton] */
```

Comment:
Not from this PR, but that one (@ link) doesn't work on Android Studio, not that I can see. If it did, it would cover only one of the two following overloads, right?